18-multitable.Rmd

```diff
@@ -1,11 +1,126 @@
+profiles quantified with different measurement technologies, such as
+phylogenetic microarrays, amplicon sequencing, or metagenomic
+sequencing. Such alternative experiments that concern the same samples
+can be stored as
+
+1. Separate _assays_ assuming that the taxonomic information can be mapped 
+between feature directly 1:1; or 
+2. data in the _altExp_ slot of the `TreeSummarizedExperiment`, if the feature 
+dimensions differ. Each element of the _altExp_ slot is a `SummarizedExperiment`
+or an object from a derived class with independent feature data.
+
+
+As an example, we show how to store taxonomic abundance tables
+agglomerated at different taxonomic levels. However, the data could as
+well originate from entirely different measurement sources as long as
+the samples are matched.
+
+```{r}
+# Agglomerate the data to Phylym level
+se.phylum <- agglomerateByRank(se, "Phylum")
+# both have the same number of columns (samples)
+dim(se)
+dim(se.phylum)
+
+# Add the new table as an alternative experiment
+altExp(se, "Phylum") <- se.phylum
+altExpNames(se)
+
+# Pick a sample subset: this acts on both altExp and assay data
+se[,1:10]
+dim(altExp(se[,1:10],"Phylum"))
+```
+
+For more details of altExp have a look at the [Intro vignette](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html) of the 
+`SingleCellExperiment` package [@R-SingleCellExperiment].
+
+
+
+## MultiAssayExperiments
+
+_**Multiple experiments**_ relate to complementary measurement types,
+such as transcriptomic or metabolomic profiling of the microbiome or
+the host. Multiple experiments can be represented using the same
+options as alternative experiments, or by using the
+`MultiAssayExperiment` class [@R-MultiAssayExperiment]. Depending on how the 
+datasets relate to each other the data can be stored as:
+
+1. Separate _altExp_ if the samples can be matched directly 1:1; or
+2. As `MultiAssayExperiment` objects, in which the connections between
+sample are defined through a `sampleMap`. Each element on the
+`experimentsList` of an `MultiAssayExperiment` is `matrix` or
+`matrix`-like object including `SummarizedExperiment` objects, and the
+number of samples can differ between the elements.
+
+
+```{r}
+#TODO: Find the right dataset to explain a non 1:1 sample relationship
+```
+
+
+For information have a look at the [intro vignette](https://bioconductor.org/packages/release/bioc/vignettes/MultiAssayExperiment/inst/doc/MultiAssayExperiment.html) of the `MultiAssayExperiment` package.  
+
+ 
+   Option   Rows (samples)    Cols (features)               Recommended  
+---------   --------------    ---------------  ------------------------
+   assays  	     match              match      Data transformations             
+   altExp            match              free    Alternative experiments   
+MultiAssay         mapping              free     Multi-omic experiments    
```

Comment:
This is the other way around. columns are samples and rows are features. SummarizedExperiment doesn't actually care, but SingleCellExperiment and TreeSummarizedExperiment do

Reply:
Oh yes, this was just a bug, fixed now.